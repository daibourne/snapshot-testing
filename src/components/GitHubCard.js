import React from 'react';
import { Card } from 'react-bootstrap';

const GitHubCard = () => {
    return (
        <Card style={{ width: '18rem' }}>
            <Card.Img varient='top' src='https://drive.google.com/file/d/1EJcjU18hsZ7vK1pmL5baKV8vET_MEU6w/view?usp=sharing' alt='Daijon Bourne'/ >
            <Card.Body>
                <Card.Title>daibourne</Card.Title>
                <Card.Text>The defination of beatuiful</Card.Text>
            </Card.Body>
        </Card>
    );
};

export default GitHubCard;