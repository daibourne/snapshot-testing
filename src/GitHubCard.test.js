import React from 'react';
import { render } from '@testing-library/react';
import GitHubCard from './components/GitHubCard';

test('GitHubCard matches snapshot', () => {
    const { asFragment } = render(<GitHubCard />);
    expect(asFragment()).toMatchSnapshot();
});