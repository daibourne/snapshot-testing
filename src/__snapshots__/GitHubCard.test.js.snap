// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`GitHubCard matches snapshot 1`] = `
<DocumentFragment>
  <div
    class="card"
    style="width: 18rem;"
  >
    <img
      alt="Daijon Bourne"
      class="card-img"
      src="https://drive.google.com/file/d/1EJcjU18hsZ7vK1pmL5baKV8vET_MEU6w/view?usp=sharing"
      varient="top"
    />
    <div
      class="card-body"
    >
      <div
        class="card-title h5"
      >
        daibourne
      </div>
      <p
        class="card-text"
      >
        The defination of beatuiful
      </p>
    </div>
  </div>
</DocumentFragment>
`;
